Source/GodotHelpers.cs

```diff
@@ -0,0 +1,38 @@
+using Godot;
+using System;
+
+namespace Arcadia
+{
+
+    public class GodotHelpers {
```

Comment:
So, some of type errors showed by magic I wasn't able to resolve by type hinting,  so I done it by executing static methods from this class as workaround, maybe they can be resolved in clj, then this helpers will be unneeded.